app/models/book.rb

```diff
@@ -0,0 +1,12 @@
+class Book < ApplicationRecord
+  devise :database_authenticatable, :recoverable, :rememberable, :validatable
```

Comment:
Delete this line and you'll be all good :)

I can see you've pattern matched from User - these modules included by devise (specifically `validatable`) are expecting Book to have an email field that it can validate.